test/detect/vue/default.txt

```diff
@@ -0,0 +1,82 @@
+<!--
+Things to verify:
+- comments (this block) are properly highlighted
+- expressions are highlighted as JS
+- <script> highlights ES2015 syntax
+- <style> highlights as CSS
+-->
+
+<template>
+  <div>
+    {{ foo * 10 + 'hi' }}
+    <span
+      v-text="foo * 10 + 'hi'"
+      :id="foo + 'baz'"
+      @click="onClick('hello')"
+      @click.prevent="onClick('hello')">
+      Hello
+    </span>
+  </div>
+</template>
+
+<template lang="pug">
+  <div>
```

Comment:
It need to be changed `div`. not `<div>`, also any other tags inside template using "pug"

Reply:
The sample has been updated but does not implementat a specific highlight for pug 🤷‍♀️

Reply:
thanks :)